app/models/user.rb

```diff
@@ -0,0 +1,33 @@
+require 'bcrypt'
+
+class User
+
+  include DataMapper::Resource
+  attr_reader :password
+  attr_accessor :password_confirmation
+
+  validates_confirmation_of :password
+  validates_length_of :username, :min => 1
+  validates_length_of :name, :min => 1
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer the new style validations `validates :column, length: value` over `validates_length_of`.